src/axom/quest/IntersectionShaper.hpp

```diff
@@ -1089,25 +1098,25 @@ class IntersectionShaper : public Shaper
       cfgf = newVolFracGridFunction();
       this->getDC()->RegisterField(fieldName, cfgf);
 
-      AXOM_PERF_MARK_SECTION("compute_free", {
-        int dataSize = cfgf->Size();
-        GridFunctionView<ExecSpace> cfView(cfgf);
+      AXOM_ANNOTATE_SCOPE("compute_free");
```

Comment:
Maybe I'm ignorant, but it makes better sense to me if `AXOM_ANNOTATE_SCOPE` comes right after the open-brace.  If it works, I guess it's a minor style point.

Reply:
I was following the existing markings, which began the annotated region here.

`AXOM_ANNOTATE_SCOPE` adds a variable whose constructor begins an annotation region and whose destructor is called at the end of the enclosing scope -- i.e. it doesn't need to be at the start of the `c++` scope. 

Having said that, we might want to adopt your suggested/implied convention within axom -- and I'm open to discussing this.

Reply:
Thanks for the explanation.  I'm fine with leaving AXOM_ANNOTATE_SCOPE where it is.